src/tesk_core/job.py

```diff
@@ -22,7 +23,14 @@ def run_to_completion(self, poll_interval, check_cancelled, pod_timeout):
         logging.debug("Creating job '{}'...".format(self.name))
         logging.debug(pprint(self.body))
         self.timeout = pod_timeout
-        self.bv1.create_namespaced_job(self.namespace, self.body)
+        try:
+            self.bv1.create_namespaced_job(self.namespace, self.body)
+        except ApiException as ex:
+            if ex.status == 409:
+                self.bv1.read_namespaced_job(self.name, self.namespace)
```

Comment:
It would be cool to have a trace of that situation in the log as well. Not as error.